### ISSUE TITLE
Few Issues and Bugs I noticed 

### ISSUE THREAD

**Issue by Skeletonpun** [ccfox/CWE-FoxFork#60]
Algeria should be released rather than Algerie if France is dismantled and Algerie should be able to become Algeria if they're communist (though apartheid Communist French Algeria is kinda funny)

West Berlin tends to stay extremely small due to the Berlin Wall and Landlocked modifiers when OTL despite that it had a population of 2 million (and East Berlin at around 1.2 million) at reunification  

Governmental/Societal corruption seems to be missing (I assume for a rework) but it results in the multinational coup chain firing more than it should as nobody has the low corruption modifier 

There's a Trump/Biden civil war event but it requires the Confederates and the Free States to already exist as American vassals (event 8015876) its tagged as 	#USA owns the Republican states but then checks OR = { is_our_vassal = CUA is_our_vassal = CSA } 

Also the American election events could be spruced up a bit as they all have a MTTH of 11 months probably to get them to fire in November but that's not really how MTTH works

**Comment by Skeletonpun** [comment]
> Hm, there should be a decision for this - its likely bugged.

There is (modern_algeria) but its just bugged to only allow democracies to do it rather than socialist republics/communist states

> Are they? Some events might be missing as I had to go through every single file individually to check for broken events (and likely still missed some). What file are they supposed to be in, if you know it off hand?

Events 1410087 through to 1410091 deal with social corruption and 1410082 to 1410086 deal with governmental corruption and both seem to be missing (in Essential Events 3 in the original mod files)

> Its a mess, and was made as a bit of a meme anyhow. It could probably be a lot better than even making sure all the events in it fire correctly.

Yeah its kinda ridiculous how much attention there is to the Trump presidency when its so ridiculously rare for it to fire whatsoever and even then the actual civil war doesn't have too much flavour. I'm tempted to redo the entirety of America's president events if I get the time but idk how many people would end up playing it at this point

**Comment by ccfox** [comment]
The issue with France and the missing events should be fixed.

As for redoing America's events - feel free! People still play the mod, and even this fork of it. I added a few extra candidates here and there, but I haven't brought it in line with the other elections I reworked yet or fully implemented the (probably overly complex) idea I had for tackling them.

**Comment by Skeletonpun** [comment]
What ideas did you have for America's events? I can't promise anything but I'm thinking I'll try reworking it and would love to see what you had in mind, also is there a Discord or anything for CWE?

**Comment by ccfox** [comment]
Yes, CWE's discord can be found here: https://discord.gg/BPJcMuN

As for my ideas, it was just to add in two other election paths in case the player or AI elect left or right. Essentially it'd be a way to include candidates that were minor historically but might've become larger, as well as to try to (where it makes sense and is realistic) ensure a continuation of ideologies that are not in every election (like progressives, for instance).

So an example would be electing the progressive Wallace in the early election event might mean the next election event is one where the player is given more left-leaning options. Haven't fully planned it out, but figured it would be a fairly straight-forward way to add some more dynamism to elections.
